src/pages/User/components/PullRequests/UserInfo/Message/Message.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import getMessage from './getMessage';
+
+const Message = ({ pullRequestCount, otherReposCount }) => (
+  <h2 className="font-medium text-red-dark text-3xl">{getMessage(pullRequestCount, otherReposCount)}</h2>
```

Comment:
The text should not be `dark red` which would signify a failure action. Instead either something more neutral like `blue/black/dark-grey` or success like `green`.